api/v1alpha1/virtualmachine_types.go

```diff
@@ -98,7 +98,7 @@ type VirtualMachineMetadataTransport string
 
 const (
 	// VirtualMachineMetadataExtraConfigTransport will set the VirtualMachineMetadata ConfigMap data as
-	// extraConfig key value fields on the VM.
+	// extraConfig key value fields on the VM. Only keys prefixed with "guestinfo." will be allowed.
```

Comment:
I'm not sure if "allowed" is the right wordage here since there is no feedback. Maybe "set" or "copied" instead?

Reply:
Ah yeah, I'll change it to "set"